compose.yaml

```diff
@@ -78,6 +78,7 @@ services:
       NEXT_PUBLIC_ENABLE_MANIFEST_ROUTE: ${NEXT_PUBLIC_ENABLE_MANIFEST_ROUTE}
       NEXT_PUBLIC_MANIFEST_FORCE_ENABLE: ${NEXT_PUBLIC_MANIFEST_FORCE_ENABLE}
       NEXT_PUBLIC_MANIFEST_ALLOWED_DOMAINS: ${NEXT_PUBLIC_MANIFEST_ALLOWED_DOMAINS}
+      MANIFEST_ROUTE_FORCE_ENABLE: "true"
```

Comment:
As @ronibhakta1 has mentioned, we don't want to hard code these "true" values; we should set one variable that then gets passed along correctly to the files that are needed. Since the `compose` file calls the `dockerfile` it makes sense that we would pass along the `MANIFEST_ROUTE_FORCE_ENABLE` variable and reference it from the dockerfile (rather than specifying `true` directly in the dockerfile). If we're going to always specify `MANIFEST_ROUTE_FORCE_ENABLE` in the dockerfile as true, then our compose doesn't need such a variable (it's duplicative and superfluous).